### ISSUE TITLE
Drag & Drop tools not working inside iframe

### ISSUE THREAD

**Issue by bodanu** [unlayer/react-email-editor#192]
I'm  using  react-email-editor rendered inside an iframe, which  is by itself rendered in another iframe as an embeded app. 
The  component renders without errors  or warnings, but none of the drag-and-drop features work.

No errors show up when dragging and dropping either.
Is this an expected behaviour being used this way? Or is it an  issue.

"react-email-editor": "^1.2.0",
 "react": "^17.0.2",
 "react-dom": "^17.0.2",

Code used:
```
import React, { useRef } from 'react';
import { render } from 'react-dom';

import EmailEditor from 'react-email-editor';

const NewMail = (props) => {
    const emailEditorRef = useRef(null);

    const exportHtml = () => {
        emailEditorRef.current.editor.exportHtml((data) => {
        const { design, html } = data;
        console.log('exportHtml', html);
        });
    };

    const onLoad = () => {
        // you can load your template here;
        // const templateJson = {};
        // emailEditorRef.current.editor.loadDesign(templateJson);
    };

    return (
        <div>
        <div>
            <button onClick={exportHtml}>Export HTML</button>
        </div>
            <EmailEditor
            ref={emailEditorRef}
            onLoad={onLoad}
            options={{}}
            />
        </div>
    );
}

export default NewMail;
```

![image](https://user-images.githubusercontent.com/13087899/116403387-2c4ee380-a836-11eb-8da5-e0e00d05f080.png)

Later edit:

This seems to happen only when using Chrome.

**Comment by damiandzwigaj** [comment]
We have similar problem but in our case unlayers editor.js file breaks every drag & drop events in our react application

**Comment by emattson** [comment]
Any update on this? I'm seeing the same issue intermittently in chrome as well

**Comment by chrismccaw** [comment]
I need an update on this as well. We cant use this until this is fixed

**Comment by Abner97** [comment]
Same problem here, Drag and Drop not working on Chrome.

**Comment by pierrelh** [comment]
Same problem here, seems also to be on Chrome and MacOS

**Comment by vcdotwine** [comment]
We are facing the same issue.
Chrome latest version.
Mac OS.

 Any workarounds or ETA on a fix?

**Comment by philnwoha** [comment]
Hi we just had a client encounter this issue, can confirm it seems to be related to chrome.  Using Safari, Edge, Firefox or Brave does not seem to encounter the issue.  Since chrome is the most popular browser I would imagine this would be relatively high priority.  Its understandable that most users would not use an iframe but there are some cases where this is needed especially when integrating with third parties.

**Comment by ckeyes88** [comment]
Has this been resolved or identified? We're seeing this as well.

**Comment by damiandzwigaj** [comment]
> Has this been resolved or identified? We're seeing this as well.

I posted the issue long time ago https://unlayer.canny.io/bug-reports/p/email-editor-affects-dragging-of-other-components-in-the-application and they marked it as complete so guess they wont fix it

**Comment by markgustav** [comment]
We're experiencing this in our app and hearing from customers who are unhappy they can't use Chrome. Can we get some acknowledgment, Unlayer, with an estimation of when it will be remedied? Thanks. 

Also, another post referencing this in your bug reporting tool: https://unlayer.canny.io/bug-reports/p/unable-to-drop-element-in-the-content-on-macos-x-chrome

**Comment by adeelraza** [comment]
@markgustav is your app inside an iframe?

**Comment by markgustav** [comment]
That may be the issue @adeelraza. I'm going to check with my dev team. We're a Shopify app and we're loading the editor inside of their admin.

**Comment by markgustav** [comment]
Yep, that's it @adeelraza. Do you have any plans to remedy this situation or is it something that can't be fixed?

**Comment by adeelraza** [comment]
@markgustav we will internally do some research again whether there's an available remedy on this. I believe this is an active bug in the chromium database. Let us know if you or anyone here finds a possible workaround / solution.

**Comment by markgustav** [comment]
Thanks @brunolemos for posting this. This workaround will be somewhat helpful until a more permanent solution can be discovered.

**Comment by brunolemos** [comment]
We have fixed this issue in version `1.5.49`.
Thanks everyone for the report!

Let me know if it's now working fine for you inside your iframes.

cc @markgustav @bodanu
